### ISSUE TITLE
i don't know, I crashed it

### ISSUE THREAD

**Issue by angelxaces** [vanowm/FirefoxCookiesManagerPlus#126]
sorry, busy, no time for error reports. running a bunch of tabs, closed a few before it crashed but I could save it.

**Comment by vanowm** [comment]
And that related to CM+ how?

**Comment by angelxaces** [comment]
it relates because it caused an error the git felt needed to be reported. since it was going to show up in this I wanted to to say the the problem was on my end so if someone went looking for bugs they would know right away not to waste time on this error. I never leave git notes blank

**Comment by vanowm** [comment]
Sorry, you totally lost me on this one...git as in github.com? Firefox crashed and not only it specifically knew what cased the crash (in this case by an addon), but it also offered you submit the crash report here?

I'm sincerely have no idea what you are talking about. Can you dumb it down it for me?

Thanks.

**Comment by angelxaces** [comment]
I'm sorry, I'm going back through stuff and realized I completely blew you off. That was an accident, I don't mean to be rude. I remember what happened here though.

At the time of the crash I was prompted with an error submission message. Often those are just asking if the user would like to add anything and will send even if they are left blank. I also made one thing very vague, sorry about that. An excessive amount of tabs  in my browser  were the cause of the crash due to hoarding memory like a packrat in paradise. But the crash didn't just take down the browser, it forced the whole computer to reboot and that meant my git GUI client as well. Since that program was improperly exited it prompted me to say something. I didn't know if it was one that would send with or without user input so I was essentially trying to say that the error was on my end and not the application. In my haste I managed to make that much more confusing and should have slowed down a bit to write in some semblance of a language, English preferably. 

Anyways, sorry for that confusion

**Comment by vanowm** [comment]
Oh, did that git GUI crash report actually created this issue??? That would be a total overkill for a script...

Anyhow, I gather this issue can be closed now?

**Comment by angelxaces** [comment]
Ya, as there never really was an issue. At the time I didn't realize it would create an issue, I thought it would mark it as a minor note of some kind. I wasn't used to using github so I had no clue how it would handle crash reporting. I guess ultimately I made an issue to say there wasn't an issue. At least not on the app's side of things.
